src/debugger/Debugger.h

```diff
@@ -115,6 +123,23 @@ class Debugger : public gc {
         ESCARGOT_RECORD_UNKNOWN_ENVIRONMENT = 5,
     };
 
+    // Variable types
+    enum {
+        ESCARGOT_VARIABLE_END = 0,
+        ESCARGOT_VARIABLE_UNACCESSIBLE = 1,
+        ESCARGOT_VARIABLE_UNDEFINED = 2,
+        ESCARGOT_VARIABLE_NULL = 3,
+        ESCARGOT_VARIABLE_TRUE = 4,
+        ESCARGOT_VARIABLE_FALSE = 5,
+        ESCARGOT_VARIABLE_NUMBER = 6,
+        ESCARGOT_VARIABLE_STRING = 7,
+        ESCARGOT_VARIABLE_OBJECT = 8,
+        ESCARGOT_VARIABLE_ARRAY = 9,
+        ESCARGOT_VARIABLE_FUNCTION = 10,
+        ESCARGOT_VARIABLE_LONG_NAME = 0x40,
+        ESCARGOT_VARIABLE_LONG_VALUE = 0x80,
+    };
```

Comment:
How many kinds of types does the debugger need to support?
The above list does not strictly follow the types in JavaScript language standard (boolean, null, undefined, number, string, symbol and object).

Reply:
This is a hint for users, rather than strict types. As for debugger IDEs (e.g. VSCode) they are just strings, they have no special meaning (from debugger perspective, everything is just strings). More types usually better for humans. If we want, we can add types for some built-in objects, such as typed Arrays. It is up to us. We can also decide how we want to show values.